Example/Firebase.xcodeproj/project.pbxproj

```diff
@@ -6631,12 +6631,12 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CLANG_ANALYZER_NONNULL = YES;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				DEBUG_INFORMATION_FORMAT = dwarf;
```

Comment:
Can you revert the project file and do those in a separate commit? They should be changed but no reason to be part of this PR :)